spec/datetimeformat.html

```diff
@@ -102,16 +97,14 @@
         1. Let _opt_ be a new Record.
         1. For each row of <emu-xref href="#table-3">Table 3</emu-xref>, except the header row, do:
           1. Let _prop_ be the name given in the Property column of the row.
-          1. Let _value_ be GetOption(_options_, _prop_, *"string"*, «the strings given in the Values column of the row», *undefined*).
-          1. ReturnIfAbrupt(_value_).
+          1. Let _value_ be ? GetOption(_options_, _prop_, *"string"*, «the strings given in the Values column of the row», *undefined*).
```

Comment:
Yes, I was confused looking at the source here because there are probably 100's of places where literal strings or intrinsics should've been formatted as code, but have only `*"foo"*` or `*foo*` and I guess I thought those were correct, but was obviously misunderstanding what I was reading. 

Compare: 

![](https://i.gyazo.com/8c3dbe348104be74bfa9f7cacba94ff6.png)

to: 

![](https://i.gyazo.com/7a667c2287d85fb9d08c718acd7a6cf9.png)